webrender/src/frame_builder.rs

```diff
@@ -1050,6 +1046,7 @@ impl FrameBuilder {
             run: TextRun {
                 font_key,
                 logical_font_size: size,
+                run_offset,
```

Comment:
There is an offset field in text runs with this https://github.com/servo/webrender/pull/1498/ - perhaps we can make use of that here, which applies the offset in the vertex shader? That could be done as a follow up though.